api/config/v1alpha1/envoyproxy_types.go

```diff
@@ -56,7 +56,7 @@ type EnvoyProxySpec struct {
 	// Bootstrap configuration used. You can edit this configuration, and rerun `egctl x translate` to ensure there are no validation errors.
 	//
 	// +optional
-	Bootstrap *string `json:"bootstrap,omitempty"`
```

Comment:
this's a breaking change?

Reply:
Yes, it's a breaking change. We can make it non-breaking by leaving the current `Booststrap` as a string and adding a new field, but I can't think of a good name for the new field. Do you have any idea?

Reply:
I'm OK with this, but must highlight in release-notes.

Reply:
yes, added the release-notes tag